web/src/Components/Delegation.elm

```diff
@@ -6,7 +6,7 @@ import Element exposing (el, text)
 import Helpers exposing (dubCol, getField, para)
 import Models exposing (Model)
 import Models.Democracy exposing (DelegateState(..), Democracy, DemocracyId)
-import Msgs exposing (Msg(AddDelegate, MultiMsg, RemoveDelegate, Send, SetDelegateState, SetField))
+import Msgs exposing (Msg(AddDelegate, BlockchainSend, MultiMsg, RemoveDelegate, SetDelegateState, SetField))
```

Comment:
I prefer `import Msgs exposing (Msg(..))` for stuff like this - avoids extra imports, enables code completion, and reduces diff size/complexity.